docs/oidc_testing/examples/config-dev.yaml.example

```diff
@@ -0,0 +1,100 @@
+# The base path of dex and the external name of the OpenID Connect service.
```

Comment:
This entire file should be made more minimal to suit our needs. All upstream comments for customization do not need to be included.

Reply:
I agree.  My original line of thinking was to leave it as close to upstream as possible to make it easier to merge future upstream changes in, but readability suffers as a result.  I'll clean that up.

Reply:
Completely restructured to be a bit more useful in 4766fbf